### PR TITLE
REF: switch from host_inventory.yaml to LCLS-style directory structure

### PR DESCRIPTION
**Pull request by klauer**
## Context

* Restructuring host_inventory.yaml to LCLS-style directory structure
* This should ease integration down the line
* Makes it easier to specify different roles and configuration for individual PLCs
* This may be excessive for "single PLC ansible example repo" - but we're looking forward here

### CODE REVIEW

**Comment by klauer** [comment]
Let's get this merged before our hackathon

File: group_vars/tcbsd_plcs/vars.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+ansible_user: Administrator
+ansible_ssh_private_key_file: /Users/klauer/Repos/twincat-bsd-ansible-testing/tcbsd_key_rsa
```
Comment (tangkong) [inline group_vars/tcbsd_plcs/vars.yml:3]: Does this (and the rest of the file, for that matter) get templated / filled in some way? Or is this file just meant to be an example?

**Review by tangkong — APPROVED** [review]

**Review by ZLLentz — APPROVED** [review]
I don't have enough experience/knowledge to understand the significance of this change but it seems good

File: group_vars/tcbsd_plcs/vars.yml (hunk 1 above)
Comment (klauer) [reply group_vars/tcbsd_plcs/vars.yml:3]: A good catch - just an example (still probably something I should have changed before asking for a review to be honest...)

**Pull request merged**